po/hu.po

```diff
@@ -227,125 +262,151 @@ msgstr "Név"
 
-#: src/exm-browse-page.blp:68
+#: src/exm-browse-page.blp:69
 msgid "No Results Found"
 msgstr "Nincs találat"
 
 #. Translators:
 #. - '%s' is an extension e.g. Blur my Shell
 #. - Please use unicode quotation marks e.g. “” (not "")
-#: src/exm-browse-page.c:292
+#: src/exm-browse-page.c:289
 #, c-format
 msgid "e.g. “%s”"
 msgstr "például „%s”"
 
-#: src/exm-comment-dialog.blp:7
+#: src/exm-comment-dialog.blp:9
 msgid "Comments"
 msgstr "Megjegyzések"
 
-#: src/exm-comment-dialog.blp:34 src/exm-detail-view.blp:93
-#: src/exm-detail-view.blp:236 src/exm-detail-view.c:390
+#: src/exm-comment-dialog.blp:36 src/exm-detail-view.blp:88
+#: src/exm-detail-view.blp:241 src/exm-detail-view.c:405
 msgid "An Error Occurred"
 msgstr "Hiba történt"
 
-#: src/exm-detail-view.blp:43
+#: src/exm-detail-view.blp:38
 msgid "Zoom In"
 msgstr "Nagyítás"
 
-#: src/exm-detail-view.blp:52
+#: src/exm-detail-view.blp:47
 msgid "Zoom Out"
 msgstr "Kicsinyítés"
 
-#: src/exm-detail-view.blp:62
+#: src/exm-detail-view.blp:57
 msgid "Return to view"
 msgstr "Visszatérés a részletekhez"
 
-#: src/exm-detail-view.blp:150
+#: src/exm-detail-view.blp:145
 msgid "Enlarge image"
 msgstr "Kép nagyítása"
 
-#: src/exm-detail-view.blp:162 src/exm-extension-row.blp:78
+#: src/exm-detail-view.blp:157 src/exm-extension-row.blp:89
 msgid "Description"
 msgstr "Leírás"
 
-#: src/exm-detail-view.blp:181
+#: src/exm-detail-view.blp:178
+msgid "Links"
+msgstr "Hivatkozások"
+
+#: src/exm-detail-view.blp:189
+msgid "Homepage"
+msgstr "Honlap"
+
+#: src/exm-detail-view.blp:201
 msgid "View on Extensions"
 msgstr "Megtekintés böngészőben"
 
-#: src/exm-detail-view.blp:198
-msgid "Supported Versions"
-msgstr "Támogatott GNOME verzió"
-
-#: src/exm-detail-view.blp:214
+#: src/exm-detail-view.blp:219
 msgid "User Reviews"
 msgstr "Felhasználói értékelések"
 
-#: src/exm-detail-view.blp:255
-msgid "Show All Reviews"
-msgstr "Összes értékelés megtekintése"
+#: src/exm-detail-view.blp:250
+msgid "There are no comments."
+msgstr "Nincsenek értékelések."
+
+#: src/exm-detail-view.blp:270
+msgid "_Show All Reviews"
+msgstr "_Összes értékelés megtekintése"
 
 #. Translators: Use unicode ellipsis '…' rather than three dots '...'
-#: src/exm-detail-view.c:404
+#: src/exm-detail-view.c:419
 msgid "Loading…"
 msgstr "Betöltés…"
 
 #: src/exm-error-dialog.blp:7
 msgid "Error Report"
 msgstr "Hiba jelentése"
 
-#: src/exm-error-dialog.blp:27
-msgid ""
-"An unexpected error occurred in Extension Manager. Please open a new issue "
-"and attach the following information:"
-msgstr ""
-"Nem várt kivétel történt a kiterjesztés-kezelőben. Kérlek, jelentsd a hibát "
-"és mellékeld az alábbiakat:"
+#: src/exm-error-dialog.blp:29
+msgid "An unexpected error occurred in Extension Manager."
+msgstr "Nem várt kivétel történt a kiterjesztés-kezelőben"
 
-#: src/exm-error-dialog.blp:50 src/exm-upgrade-assistant.blp:127
+#: src/exm-error-dialog.blp:56 src/exm-upgrade-assistant.blp:127
 msgid "Copy to Clipboard"
```

Comment:
The original message is missing the underscore, which is strange since the template has it. See
https://github.com/mjakeman/extension-manager/blob/4edb96810c2f58ba4d6f7129d7c1483e0ba54e79/po/extension-manager.pot#L341
You could try to update from the template or add it by hand.